inst/qml/LSBuffonsneedlemanipulation.qml

```diff
@@ -127,7 +133,56 @@ Form
 			}
 		}
 
+		Group
```

Comment:
should this group be inside of the checkbox for the prior and posterior for the proportion of pi, since it affects only that plot?

Reply:
yes, I moved that group inside the checkbox "implied prior and posterior for the proportion of pi".